GrindingCity.WebApi/Repositories/ResourseRepository.cs

```diff
@@ -0,0 +1,57 @@
+﻿using GrindingCity.WebApi.DTOs;
+using GrindingCity.WebApi.Interfaces;
+using GrindingCity.WebApi.Models;
+using Microsoft.EntityFrameworkCore;
+
+namespace GrindingCity.WebApi.Repositories
+{
+    public class ResourseRepository : IResourseRepository
+    {
+        private AppDbContext _appDbContext;
+
+        public ResourseRepository(AppDbContext db) 
+        {
+            _appDbContext = db;
+        }
+        
+        public async Task<Resourse?> GetResourse(Guid id) => await _appDbContext.Resourses.FirstOrDefaultAsync(b => b.Id == id);
+        
+        public async Task<IEnumerable<Resourse>> GetAllResourses() => await _appDbContext.Resourses.ToListAsync();
+
+        public async Task AddResourse(CreateResourseRequest request)
+        {
+            var newResourse = new Resourse()
+            {
+                Title = request.title,
+                Price = request.price
+            };
+
+            _appDbContext.Resourses.Add(newResourse);
+            await _appDbContext.SaveChangesAsync();
+        }
+        
+        public async Task UpdateResourse(Guid id, CreateResourseRequest request)

```

Comment:
`CreateResourseRequest` a little confusing for update. I would portably introduce another type or rename existing